client/login/wp-login/style.scss

```diff
@@ -297,6 +297,14 @@ $image-height: 47px;
 		padding: 0;
 	}
 
+	.step-container-v2__content-wrapper {
```

Comment:
Let's not add StepContainerV2 CSS overrides, please.

Reply:
The design has a bit of space between the top bar and the heading. According to Francisco's comment above, it's meant to be vertically aligned

> The whole content area is not vertically aligned to balance the top and bottom spaces according to the browser window, leaving an empty space at the bottom.

How should I go about doing this using vanilla StepContainerV2?

Reply:
Ah, that works, thanks!